examples/messengers/telegram/3_responses_with_media.py

```diff
@@ -0,0 +1,124 @@
+# %% [markdown]
+"""
+# 3. Responses with Media
+
+This example demonstrates how to configure your bot to exchange media
+using the generic `Response` class from DFF.
+"""
+
+# %%
+import os
+
+import dff.script.conditions as cnd
+from dff.script import Context, Actor, TRANSITIONS, RESPONSE
+
+from telebot import types
+
+from dff.messengers.telegram import (
+    PollingTelegramInterface,
+    TelegramMessenger,
+    update_processing_service,
+)
+from dff.pipeline import Pipeline
+from dff.script.responses.generics import Response, Image
+from dff.utils.testing.common import is_interactive_mode
+
+
+# %%
+# kitten picture info:
+kitten_id = "Y0WXj3xqJz0"
+kitten_ixid = "MnwxMjA3fDB8MXxhbGx8fHx8fHx8fHwxNjY4NjA2NTI0"
+kitten_width = 640
+kitten_url = f"https://unsplash.com/photos/" f"{kitten_id}/download?ixid={kitten_ixid}" f"&force=true&w={kitten_width}"
+
+
+# %% [markdown]
+"""
+To detect media, write a function that processes Telebot types, like `Message`.
+This function will be passed to `message_handler` in the script.
+"""
+
+
+# %%
+def doc_is_photo(message: types.Message):
+    return message.document and message.document.mime_type == "image/jpeg"
+
+
+# %% [markdown]
+"""
+To send media, instantiate the generic `Response` class with the property `image`
+set to `Image` class. For the latter, you can use both online media files and
+local media files on your computer. The same applies to audio, video, documents and
+other types of media.
+"""
+
+
+# %%
+# Like Telebot, TelegramMessenger only requires a token to run.
+# However, all parameters from the Telebot class can be passed as keyword arguments.
+messenger = TelegramMessenger(os.getenv("TG_BOT_TOKEN", "SOMETOKEN"))
+
+
+# %%
+script = {
+    "root": {
+        "start": {RESPONSE: Response(text=""), TRANSITIONS: {("pics", "ask_picture"): cnd.true()}},
```

Comment:
I think, it should be done for all cases

Reply:
If we make the /start command obligatory to start the dialog, the bot will reply with a fallback option in case the user sends something else, which will look ugly. Instead of /start the user can write anything else